src/clauses/Union.test.ts

```diff
@@ -42,4 +42,26 @@ describe("CypherBuilder UNION", () => {
         `);
         expect(queryResult.params).toMatchInlineSnapshot(`{}`);
     });
+
+    test("Union ALL Movies", () => {
+        const n1 = new Cypher.Node({ labels: ["Movie"] });
+        const query1 = new Cypher.Match(n1).return(n1);
+        const n2 = new Cypher.Node({ labels: ["Movie"] });
+        const query2 = new Cypher.Match(n2).return(n2);
+        const n3 = new Cypher.Node({ labels: ["Movie"] });
+        const query3 = new Cypher.Match(n3).return(n3);
+
+        const unionQuery = new Cypher.Union(query1, query2, query3).all();
+        const queryResult = unionQuery.build();
+        expect(queryResult.cypher).toMatchInlineSnapshot(`
```

Comment:
This example is not right, you have to return the same columns in every query

Reply:
Good catch